README.md

```diff
@@ -1,3 +1,15 @@
 # marta-ga-us
 
-Information about MARTA (Atlanta, GA - http://www.itsmarta.com/).
+Tracking issues for [MARTA](http://www.itsmarta.com/) (Atlanta, GA) GTFS and GTFS-realtime data.
```

Comment:
I was originally planning on putting full version of MARTA spelled out, but I looked at their website and the full spelling doesn't appear anywhere.  So I was thinking they are just branding themselves as "MARTA" now?  I agree though that spelling it out would help - I'll do that here.